draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -820,79 +751,29 @@ This OID might or might not be used as the final OID for the
 extension, so certificates including it might need to be reissued once the final
 RFC is published.
 
-# Transferring a signed exchange {#transfer}
-
-A signed exchange can be transferred in several ways, of which three are
-described here.
-
-## Same-origin response {#same-origin-response}
-
-The signature for a signed exchange can be included in a normal HTTP response.
-Because different clients send different request header fields, and intermediate
-servers add response header fields, it can be impossible to have a signature for
-the exact request and response that the client sees. Therefore, when a client
-calls the validation procedure in {{signature-validity}}) to validate the
-`Signature` header field for an exchange represented as a normal HTTP
-request/response pair, it MUST pass:
-
-* The `Signature` header field,
-* The effective request URI (Section 5.5 of {{!RFC7230}}) of the request,
-* The serialized headers defined by {{serialized-headers}}, and
-* The response's payload.
-
-If the client relies on signature validity for any aspect of its behavior, it
-MUST ignore any header fields that it didn't pass to the validation procedure.
-
-### Serialized headers for a same-origin response {#serialized-headers}
-
-The serialized headers of an exchange represented as a normal HTTP
-request/response pair (Section 2.1 of {{?RFC7230}} or Section 8.1 of
-{{?RFC7540}}) are the canonical serialization ({{canonical-cbor}}) of the CBOR
-representation ({{cbor-representation}}) of the following request and response
-metadata and headers:
-
-* The method (Section 4 of {{!RFC7231}}) of the request.
-* The response status code (Section 6 of {{!RFC7231}}) and the response header
-  fields whose names are listed in that exchange's `Signed-Headers` header field
-  ({{signed-headers}}). If a response header field name from `Signed-Headers`
-  does not appear in the exchange's response header fields, the exchange has no
-  serialized headers.
-
-If the exchange's `Signed-Headers` header field is not present, doesn't parse as
-a Structured Header ({{!I-D.ietf-httpbis-header-structure}}) or doesn't follow
-the constraints on its value described in {{signed-headers}}, the exchange has
-no serialized headers.
+Some certificates have already been issued with this extension and with validity
+periods longer than 90 days. These certificates will not immediately be treated
+as invalid. Instead:
 
-### The Signed-Headers Header {#signed-headers}
+* Clients MUST reject certificates with this extension that were issued after
+  2019-05-01 and have a Validity Period longer than 90 days.
+* After 2019-08-01, clients MUST reject all certificates with this extension
+  that have a Validity Period longer than 90 days.
```

Comment:
Update line 636 so that `main-certificate`'s validity period is checked too?

BTW Chrome doesn't implement this yet. I'll try to implement this in M75. https://crbug.com/953165

Reply:
Done.